demo/backend/ui/styling/simple-ui-kit/_uikit.xml.njk

```diff
@@ -691,17 +691,17 @@
 <style id="shadow-footer" backgroundColor="white" shadowRadius="16" shadowOffsetY="-3.6"/>
 <style id="shadow-pressable">
   <modifier pressed="true">
-    <style elevation="1" shadowOffsetY="1" shadowRadius="1"/>
+  <style elevation="1" shadowOffsetY="1" shadowRadius="1"/>
```

Comment:
Just checking, is this the intended style? Does it have to do with self-closing tags?

Reply:
I think the linter was designed primarily for html and might not handle every tags correctly